tools/setup.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Use this script to create a wheel with OpenVINO™ Python* tools:
+
+$ python setup.py sdist bdist_wheel
+"""
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r", encoding="utf-8") as f:
+    long_description = f.read()
+
+with open('benchmark/requirements.txt') as f:
```

Comment:
how about windows paths with '\'

Reply:
This works on Windows, since path-like object is expected.